src/mahoji/commands/testpotato.ts

```diff
@@ -0,0 +1,342 @@
+						}
+					]
+				},
+				{
+					type: ApplicationCommandOptionType.Subcommand,
+					name: 'setxp',
+					description: 'Set skill kc.',
+					options: [
+						{
+							type: ApplicationCommandOptionType.String,
+							name: 'skill',
+							description: 'The skill.',
+							required: true,
+							choices: Object.values(Skills).map(s => ({ name: s.name, value: s.id }))
+						},
+						{
+							type: ApplicationCommandOptionType.Integer,
+							name: 'xp',
+							description: 'The xp you want.',
+							required: true,
+							min_value: 1,
+							max_value: 200_000_000
+						}
+					]
+				},
+				{
+					type: ApplicationCommandOptionType.Subcommand,
+					name: 'setminigamekc',
+					description: 'Set minigame kc.',
+					options: [
+						{
+							type: ApplicationCommandOptionType.String,
+							name: 'minigame',
+							description: 'The minigame you want to set your KC for.',
+							required: true,
+							autocomplete: async value => {
+								return Minigames.filter(i => {
+									if (!value) return true;
+									return [i.name.toLowerCase(), i.aliases].some(i => i.includes(value.toLowerCase()));
+								}).map(i => ({
+									name: i.name,
+									value: i.column
+								}));
+							}
+						},
+						{
+							type: ApplicationCommandOptionType.Integer,
+							name: 'kc',
+							description: 'The minigame KC you want.',
+							required: true,
+							min_value: 0,
+							max_value: 10_000
+						}
+					]
+				},
+				{
+					type: ApplicationCommandOptionType.Subcommand,
+					name: 'reset',
+					description: 'Totally reset your account.'
+				},
+				{
+					type: ApplicationCommandOptionType.Subcommand,
+					name: 'gear',
+					description: 'Spawn food, pots, runes, coins, blowpipe, POH with a pool, and BiS gear.'
+				},
+				{
+					type: ApplicationCommandOptionType.Subcommand,
+					name: 'max',
+					description: 'Set all your stats to the maximum level, and get max QP.'
+				},
+				{
+					type: ApplicationCommandOptionType.Subcommand,
+					name: 'patron',
+					description: 'Set your patron perk level.',
+					options: [
+						{
+							type: ApplicationCommandOptionType.String,
+							name: 'tier',
+							description: 'The patron tier to give yourself.',
+							required: true,
+							choices: [
+								{
+									name: 'Non-patron',
+									value: '0'
+								},
+								{
+									name: 'Tier 1',
+									value: '1'
+								},
+								{
+									name: 'Tier 2',
+									value: '2'
+								},
+								{
+									name: 'Tier 3',
+									value: '3'
+								},
+								{
+									name: 'Tier 4',
+									value: '4'
+								},
+								{
+									name: 'Tier 5',
+									value: '5'
+								}
+							]
+						}
+					]
+				}
+			],
+			run: async ({
+				options,
+				userID
+			}: CommandRunOptions<{
+				max?: {};
+				patron?: { tier: string };
+				gear?: {};
+				reset?: {};
+				setminigamekc?: { minigame: string; quantity: number };
```

Comment:
Quantity doesn't exist, the field is kc so it ends up passing an undefined value